lib/workers/branch/commit.ts

```diff
@@ -34,6 +34,18 @@ export function commitFilesToBranch(
   // istanbul ignore if
   if (getAdminConfig().dryRun) {
     logger.info('DRY-RUN: Would commit files to branch ' + config.branchName);
+    logger.info('DRY-RUN: Commit message would be:\n' + config.commitMessage);
+    logger.info(
+      [
+        'DRY-RUN: Committed files would be:',
+        // Piping f.contents to
+        // git diff -- ${f.name} -
+        // would be much better, but this seems to be impossible with simple-git
+        ...updatedFiles.map(
+          (f) => `${f.name}\n<<<\n${String(f.contents)}\n>>>\n`
+        ),
+      ].join('\n')
+    );
```

Comment:
```suggestion
    logger.info({
      commitMessage: config.commitMessage,
       updatedFiles: ...updatedFiles.map(
          (f) => ({name: f.name, contents: f.contents})
        ),
      },
      'DRY-RUN: Would commit files to branch ' + config.branchName),
    );
```

Use logger meta instead of logging a huge string message. Fell free to optimize the meta to your needs

Reply:
I've put the formatted output inside meta. The default sanitized output is hard to read for file contents that contain line-breaks.

What do you think about the workarounds to get a `git diff` here?